tests/app/conftest.py

```diff
@@ -636,8 +636,24 @@ def service_cleanup(  # noqa: C901
         # Clear service_letter_contacts
         session.execute(delete(ServiceLetterContact).where(ServiceLetterContact.service_id == service_id))
 
-        # Clear template_folder
+        # Clear template_folder and template_reacted object
```

Comment:
Update to teardown objects properly being created by "sample" objects (ie `sample_service`, `sample_template`...)